zoo-css-challenge/style.css

```
@@ -178,6 +184,11 @@ footer {
 .membership {
   text-align: center;
   background-color: #fffffc;
+  max-width: 700px;
+  margin-top: 100px;
+  margin-bottom: 200px;
+  margin-right: 200px;
+  margin-left: 80px;
```

```suggestion
  margin: auto;
```
We can use `margin: auto` to centre the element. `auto` distributes the margin space equally